src/Tutor.Core/InstructorModel/InstructorService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using FluentResults;
+using Tutor.Core.DomainModel.KnowledgeComponents;
+using Tutor.Core.LearnerModel.DomainOverlay.EnrollmentModel;
+
+namespace Tutor.Core.InstructorModel;
+
+public class InstructorService : IInstructorService
+{
+    private readonly IInstructorRepository _instructorRepository;
+
+    public InstructorService(IInstructorRepository instructorRepository)
+    {
+        _instructorRepository = instructorRepository;
+    }
+    
+    public Result<List<Course>> GetCourses(int instructorId)
```

Comment:
The service method should reflect the intent - we are "GettingOwnedCourses", not any courses.